### ISSUE TITLE
Allow New Chapters to be Added on Allegations Page

### ISSUE THREAD

**Issue by andrew-rist-tw** [PublicDataWorks/complaint-manager#272]
Blocked by #271 

### Business Context:
New Orleans would like to be able to add new NOPD chapters when necessary. We will do that via a `CreatableDropdown` when adding the chapter to an officer allegation

**As an** NOIPM employee
**I want** to be able to add new NOPD chapters when adding an officer allegation
**So that** I don't have to do a workaround when there's a to wit chapter I want to add that isn't in the list

### In Scope

1. Update `officers-allegations` handler to accept chapters that weren't already created (by creating the chapter and linking it to the allegation)
2. Change Chapter dropdown on the allegations page into a `CreatableDropdown` that lets the user add new ones
3. Add trailing asterisks to labels of `Severity` and `allegation details` (e.g. "Severity *" and "Enter narrative details pertaining to this allegation *") to show that they're required while chapter isn't

### Out of Scope

1.
2.
3.

### Open Questions

1.
2.
3.

## Acceptance Criteria

1. Scenario:
   Given
   When
   Then

2. Scenario:
   Given
   When
   Then

### Checklist (should be reviewed before grooming as the BA or Pair are writing the card—NOT reviewed for the first time at grooming)

- [ ] Ensure to include all necessary mockups to this card.
- [ ] Discuss what success/failure user feedback messages would be present, if any occur (front end/back end).
- [ ] Discuss Feature Flag: Where could a development flag go to flag off incomplete work? Do we want to keep the flag long-term?
- [ ] If this card is part of a sequence of cards, discuss if this card should be blocked until the testing of previous cards in the sequence have been finished.
- [ ] Have we added edge cases as ACs that we're considered from a QA perspective
- [ ] Should the new functionalities be offered only to users with certain permissions?
- [ ] How should this functionality differ between different orgs, and if it does differ, what is the hingepoint we should use for that functionality?

**Notes:**

**Comment by andrew-rist-tw** [comment]
[chapters.csv](https://github.com/PublicDataWorks/complaint-manager/files/11436157/chapters.csv)
